test-lib.sh

```diff
@@ -1069,4 +1054,45 @@ ct_test_app_dockerfile() {
   return $ret
 }
 
+# _ct_check_testcase_result
+# -----------------------------
+# Check if testcase ended in success or error
+# Argument: result - testcase result value
+# Uses: $TESTCASE_RESULT - result of the testcase
+# Uses: $IMAGE_NAME - name of the image being tested
+_ct_check_testcase_result() {
```

Comment:
```suggestion
ct_check_testcase_result() {
```
As far as I know, we do not use `_` prefix in functions.

Reply:
Ok.